src/views/Maintenancepage.vue

```diff
@@ -0,0 +1,81 @@
+            />
+            <div class="maintenance-page_description level is-mobile">
+                <div class="maintenance-page_title">
+                    {{ $t('views.maintenance.title') }}
+                </div>
+                <div class="maintenance-page_subtitle">
+                    {{ $t('views.maintenance.subtitle') }}
+                </div>
+            </div>
+        </div>
+    </main>
+</template>
+
+<script lang="ts">
+import { Component, Vue } from 'vue-property-decorator';
+import mainStore from '@/store/main-store/MainStore';
```

Comment:
@gonzalo-plaza Why are you calling the store? and on the code, you declare the store, and declare currentLanguaje, but... where are you using currentLanguaje? this get method onli return es or en, if the language is english or spanish, but you need change this languaje,  for this, we have a generic component called 'ButtonTranslation' on login dir from appointment system, maybe you cand copy and paste this component in common dir, and use this component like is using in login view from appointments

To use this component you can see how we use this componennt on MainLoginForm componentthere are a component